### PR TITLE
Automated wheel build

### PR DESCRIPTION
**Pull request by mmulich**
This builds the wheel for linux and macos as well as the source distribution. It then publishes the results to PyPI.

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -1,10 +1,10 @@
 #!/usr/bin/env python
-# -*- coding: utf-8 -*-
-from __future__ import absolute_import, division, print_function
```
Comment (bluemellophone) [inline setup.py:3]: I'd like to keep these `__future__` imports for the time-being codebase-wide

File: CMakeLists.txt [hunk 2]
```diff
@@ -34,13 +34,13 @@ message(STATUS "PYFLANN_VERSION = ${PYFLANN_VERSION}")
 DISSECT_VERSION()
 GET_OS_INFO()
 
-if (OS_IS_MACOS)
```
Comment (bluemellophone) [inline CMakeLists.txt:37]: Pending resolution here for Macports vs Homebrew: https://github.com/WildbookOrg/wbia-vtool/pull/4#discussion_r444454154

**Review by bluemellophone — COMMENTED** [review]
Pending decision from discussion on Homebrew vs MacPorts in wbia-vtool

**Comment by mmulich** [comment]
Closed in favor of a new PR which will include the use of macports.

**Pull request closed without merging**